patches/0013-p256-armv8-optimizations

```diff
@@ -593,6 +595,19 @@ Index: aws-lc/third_party/boringssl/crypto/fipsmodule/ec/p256.c
 +  }
 +}
 +
++static int ecp_nistz256_scalar_to_montgomery_inv_vartime(const EC_GROUP *group,
```

Comment:
I don't think we have settled how deep in the code we should check for NULL pointers. What do you think?

Reply:
Same comment as https://github.com/awslabs/aws-lc/pull/57/#discussion_r516216015